packages/@divvi/mobile/src/components/FilterChipsCarousel.tsx

```diff
@@ -61,20 +61,28 @@ function FilterChipsCarousel<T>({
       testID="FilterChipsCarousel"
     >
       {chips.map((chip) => {
+        const { backgroundColor, borderColor, contentColor } = chip.isSelected
+          ? {
+              backgroundColor: Array.isArray(Colors.buttonPrimaryBackground)
+                ? Colors.buttonPrimaryBackground[0]
+                : Colors.buttonPrimaryBackground,
+              borderColor: Colors.buttonPrimaryBorder,
+              contentColor: Colors.buttonPrimaryContent,
+            }
+          : {
+              backgroundColor: Colors.buttonSecondaryBackground,
+              borderColor: Colors.buttonSecondaryBorder,
+              contentColor: Colors.buttonSecondaryContent,
+            }
+
         return (
           <View
             key={chip.id}
             style={[
               styles.filterChipBackground,
-              chip.isSelected
-                ? {
-                    backgroundColor: Colors.buttonPrimaryBackground,
-                    borderColor: Colors.buttonPrimaryBorder,
-                  }
-                : {
-                    backgroundColor: Colors.buttonSecondaryBackground,
-                    borderColor: Colors.buttonSecondaryBorder,
-                  },
+              // borderColor maybe undefined in some button configurations (e.g.,
+              // gradients), so we default to backgroundColor
```

Comment:
Otherwise with borderWidth always set to 1, this defaults to black. If we set borderWidth to 0 conditionally like buttons, the chips move by a pixel as elements are selected and unselected